.github/workflows/release-cli.yaml

```diff
@@ -3,7 +3,7 @@ name: Release CLI
 on:
   push:
     branches:
-      - main
+      - '**'  # All branches
```

Comment:
Could anyone create a branch and get a preview release to distribute? Should we check if they're part of an org?

Reply:
I am not saying make them private. 
I am saying push them to another repo so that end users don't see them and can't install them by mistake and we don't use more of our quota for hisotry which is already quite small

Reply:
If they can increase then awesome.
At some point they wanted to allow orgs to pay for space which we are happy to do. But we are still on the waiting list I believe

Reply:
YEah I think splitting it makes sense.
Ship what works and wait on what we don't have